R_scripts/weatherdat.R

```diff
@@ -0,0 +1,17 @@
+# Process and analyze weather station data
+# Stephanie Pennington | Created July 2018
+
+library(readr)
+library(dplyr)
+library(ggplot2)
+library(ggrepel)
+
+read_weather_data <- function(path) {
+  files <<- list.files(path, pattern = ".csv", full.names = TRUE)
+  list <- list()
+  for (i in files) {
+    list[[i]] <- read_csv(i)
```

Comment:
Might add a `, col_types = "......"` argument here. This eliminates the red message `read_csv` will print otherwise, and ensure we know exactly what the types are coming in.